src/crystal/system/time.cr

```diff
@@ -0,0 +1,16 @@
+module Crystal
+  # :nodoc:
+  module System
+    # :nodoc:
+    module Time
+      # Returns the number of seconds that you must add to UTC to get local time.
+      # *seconds* are absolutes.
```

Comment:
When you say "from 0" it's not at all clear you're referring to the Gregorian epoch. I'd say something like `*seconds* are measured from the Gregorian epoch (01 January 0001 00:00:00)`